scalalib/src/ScalaModule.scala

```diff
@@ -2,9 +2,8 @@ package mill
 package scalalib
 
 import scala.annotation.nowarn
-
 import mill.define.{Command, Sources, Target, Task}
-import mill.api.{DummyInputStream, PathRef, Result, internal}
+import mill.api.{DummyInputStream, Loose, PathRef, Result, internal}
```

Comment:
It looks like `Loose` isn't being used unless I'm missing it.

Reply:
Good catch. IJ always sneaks it in, when I let it add return types.